lib/utils/regexes.mjs

```diff
@@ -5,6 +5,7 @@ export const atRuleRegexes = {
 	keyframes: /^(-(o|moz|ms|webkit)-)?keyframes$/i,
 	property: /^property$/i,
 	import: /^import$/i,
+	syntax: /^syntax$/i,
```

Comment:
```suggestion
	syntaxName: /^syntax$/i,
```

For clarity, let's also move this to a new `descriptorRegexes` object.

I'll also open an issue to refactor the other names for the same reason.

Reply:
That's nice. Fixed in this commit and approved your PR.
[Refactor regex name](https://github.com/stylelint/stylelint/pull/8331/commits/e6e53d42af8af327cc89bf69eb4a83216df45da6)